src/index.html

```diff
@@ -17,11 +17,350 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="#logo-Moyo"
```

Comment:
The `href` attribute value `#logo-Moyo` seems to be a fragment identifier that doesn't correspond to any element with an `id` of `logo-Moyo` in the document. Ensure that this `href` value points to a valid target.